DECISION-MAKING.md

```diff
@@ -6,6 +6,13 @@ This document list which team or people are **responsible for making decisions**
 
 All decisions are taken by consulting various parties of different teams in and outside of Mozilla.
 
+Owners are responsible for
+
+- Providing road maps for their module that align with Mozilla OKRs
+- Breaking down tasks associated with Objectives and Key Results into sub tasks that can be worked on in a sprint
+- Making decisions about the module that align with OKRs and technical considerations
+- Raising issues associated with OKRs, milestones, tech stack etc to IAM stakeholders
```

Comment:
should we list the stakeholders?

Reply:
Might be useful as well as what role a stakeholder plays? (i.e. budget, resourcing, political capital, tie-breaking decision making)

Reply:
if anything as informational. Since you mention it, this is a space (Stackholders) where RACI/RASCI/etc make some sense though not necessarily these exact letters ;-) - while module-level/service-level/KR-level RACI makes things harder because its a lot of process

Reply:
we could add these as a separate PR for clarity

Reply:
yeah, separate PR would be best.